onnxruntime/core/providers/ngraph/ngraph_custom_op.cc

```diff
@@ -80,7 +82,36 @@ void NGRAPHCustomOp::Initialize(const OrtCustomOpApi* api, OrtKernelContext* con
     uniq_input_shape.append(reinterpret_cast<const char*>(tensor_shape.data()), ndim * sizeof(int64_t));
   }
 
-  auto it = ng_exe_map_.insert({uniq_input_shape, nullptr});  //TODO: Limit the size of map with configurable size.
+  // Get cache size from environment
+  std::string tempSize;
+  if (std::getenv("NGRAPH_EP_LRU_CACHE_SIZE")) {
+    tempSize = std::getenv("NGRAPH_EP_LRU_CACHE_SIZE");
```

Comment:
can we rename it ONNXRUNTIME_NGRAPH_LRU_CACHE_SIZE ? 
(to be consistent with other env variables such as ONNXRUNTIME_NGRAPH_DUMP_OPS)

Reply:
Sure. We also did more analysis and found increasing the default size to 294 accommodates our largest model. Will that work for you, or are there any reasons from your side to make it even larger?

Reply:
Just wanted to strike a balance between future-proofing and not having too large a size.
Maybe set it to 500 for now?

Reply:
500 gives some extra room, that works for me. The latest commit addresses this.